packages/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser.php

```diff
@@ -101,9 +108,11 @@ private function resolveAnnotationValues(BetterTokenIterator $tokenIterator): ar
     }
 
     /**
-     * @return bool|int|mixed|mixed[]|string
+     * @return array<mixed>
      */
-    private function parseValue(BetterTokenIterator $tokenIterator)
+    private function parseValue(
+        BetterTokenIterator $tokenIterator
+    ): CurlyListNode | string | array | ConstExprFalseNode | ConstExprTrueNode | ConstExprIntegerNode | DoctrineAnnotationTagValueNode
```

Comment:
The `@return array<mixed>` suggest an unclear array.
But return type shows much more. I think the `@return` annotation should be removed, as it does not add any value here. The `mixed[]` and `array` have the same information value

Reply:
Without `array` doc with type mixed specified, it make PHPStan error notice in previously https://github.com/rectorphp/rector-src/runs/2774592028#step:5:44

Do you have idea?

Reply:
I wish there was some extension that ignores this message if type is union type.

Reply:
It seems we can do with: 

```neon
# phpstan.neon
parameters:
	checkMissingIterableValueType: false
```

https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

what do you think?

Reply:
@TomasVotruba I've set `checkMissingIterableValueType: false` to `phpstan.neon` https://github.com/rectorphp/rector-src/pull/184/commits/db92d4d9e12525ffbbe20182ac0270c12f81a761

Reply:
That would disable it for every array, even

```php
function getItems(): array
{
   // ...
}
```

Reply:
Ok, reverted back. Do you have idea to handle that?

Reply:
Maybe open a upstream phpstan issue?